src/components/organize/tasks/TaskStatusChip.tsx

```diff
@@ -20,7 +20,9 @@ const TaskStatusChip: React.FunctionComponent<TaskStatusChipProps> = ({ status }
     return (
         <Chip
             label={ intl.formatMessage( { id: `misc.tasks.statuses.${status}` }) }
+            size="small"
             style={{ backgroundColor: ChipColors[status] }}
+            variant="outlined"
```

Comment:
why not do this size and variant thing in the theme overrides? with the defaultProps property. don't think we're using chips anyway anywhere else now

Reply:
Fair point. Not sure if we need chips anywhere else. I would just leave it until we use chips elsewhere and see what chip style we really want (should ask gemma perhaps.)